plugins/zone.cpp

```diff
@@ -1489,6 +1493,8 @@ command_result df_zone (color_ostream &out, vector <string> & parameters)
     bool cagezone_assign = false;
     bool nick_set = false;
     string target_nick;
+    bool enum_nick = true;
```

Comment:
I suspect this needed to be initialized to `false` - see #1709. I'm working on a fix, but this looks like a likely cause at the moment.